### PR TITLE
2nd Localization Crusade

### PR DESCRIPTION
**Pull request by arithon**
## English changelog:
- Improved the Drogbar description better

## German changelog:
- Added nightborne cultural ruler titles
- Corrected encoding of german files to UFT-8-BOM
- Religious files translated
- New bookmark and draenor event file translated
- Game concept files translated
- Basically all files are translated into german now (though there might be some updates coming regarding the titles)

## French changelog:
- Warchief now uses correct articles (french)

## Developer changelog:
- There's a couple (rather small) changes that were done to the English texts as well, please confirm they are ok.

# How to test:

### CODE REVIEW

File: common/customizable_localization/00_FR_custom_loc.txt [hunk 1]
```diff
@@ -5947,6 +5947,12 @@ FR_le_Char = {
 			OR = {
 				primary_title.tier >= tier_barony
 				any_spouse = { primary_title.tier >= tier_barony }
+
+				#Warcraft
+				AND = {
+					primary_title.tier >= tier_empire
+					primary_title:e_horde
+				}
```
Comment (zumbak04) [inline common/customizable_localization/00_FR_custom_loc.txt:5955]:
```suggestion
				primary_title = title:e_horde
```

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: localization/english/event_localization/wc_draenor_events_l_english.yml [hunk 2]
```diff
@@ -40,26 +40,26 @@
...
+ WCDRA_1005_DESC_killed:0 "Not all were so fortunate, in the end. [draenor_killed.GetFullNameNicknamed] paid the ultimate price for this victory and will be remembered with honor. Alas, there is no time to mourn, for we must press on if their sacrifice is to have any meaning."
  empty_line:0 "\n\n"
  WCDRA_1005_DESC_ending:0 "Before he was slain by [wise_mage.GetFullNameNicknamed], the ogre magi Dentarg revealed that the Elder Shaman seeks to open new portals to other worlds for the Horde to invade!\n\n[wise_mage.GetFirstName] believes that the Dark Portal could be sealed forever if we could procure the Book of Medivh from Ner’zhul himself, as well as the Skull, which is currently held by the Black Dragon Aspect, Deathwing. The Destroyer seems to be in league with the Horde - we will have to defeat him to claim the Skull!"
  THEY_GOT_ME:0 "#emp They got me...#!"
  WCDRA_1005_OPT_A_dragon:0 "Neltharion’s treachery knows no bounds!"
```
Comment (arithon) [inline localization/english/event_localization/wc_draenor_events_l_english.yml:58]: Shouldn't we use either Neltharion OR Deathwing in one event, instead of both names? (Suggested by panpoleon)

Comment (zumbak04) [reply localization/english/event_localization/wc_draenor_events_l_english.yml:58]: > Shouldn't we use either Neltharion OR Deathwing in one event, instead of both names? (Suggested by panpoleon)

It's an option for dragons. They know his real name and thus refer to him as Neltharion.

File: localization/english/wc_buildings_l_english.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ﻿l_english:
  building_type_dark_portal_01:0 "Dark Portal"
- building_type_dark_portal_01_desc:1 "A gateway that binds the two worlds: Azeroth and Draenor, allowing alien creatures to go back and forth."
+ building_type_dark_portal_01_desc:1 "A gateway that binds the two worlds of Azeroth and Draenor, allowing alien creatures to go back and forth."
  building_dark_portal_01:1 "Dark Rift"
- building_dark_portal_01_desc:1 "A tiny rift in the fabric of the universe that still connects the two worlds."
+ building_dark_portal_01_desc:1 "A tiny rift in the fabric of the universe that still connects the two worlds of Azeroth and Draenor."
  building_dark_portal_02:1 "Dark Portal"
- building_dark_portal_02_desc:1 "A massive arch with magical runes that maintains the connection between the two worlds, allowing large armies to pass through the portal."
+ building_dark_portal_02_desc:1 "A massive arch with magical runes that maintains the connection between the two worlds of Azeroth and Draenor, allowing large armies to pass through the portal."
```
Comment (robmart) [inline localization/english/wc_buildings_l_english.yml:7]: The phrase "the two world of Azeroth and Draenor" is used too often in this file, would recommend just using "between Azeroth and Draenor" or "between the worlds of Azeroth and Draenor" in at least one the these lines

File: localization/english/wc_traits_l_english.yml [hunk 4]
```diff
@@ -161,16 +161,19 @@
...
  
  trait_creature_bronze_dragon:0 "Bronze Dragon"
  trait_creature_bronze_dragon_desc:0 "Bronze dragons are smaller and more agile than dragons of the other flights, and they appear to be built for speed rather than stamina. Their bodies are thin and dexterous, with scales almost metallic in appearance. The quality of their scales is such that they are sometimes mistaken for constructs or detailed statuary. Bronze dragons are fastidious, keeping their mesh-like scales clean and shining at all times. The dragon is long and slender, its scales glinting with a metallic sheen. A spiky bone ridge runs down its back and forms a wicked point at the end of its tail. Its form shimmers from golden yellow to rich orange in the sunlight as it darts through the air. Bronze dragons prefer to use their spell-like abilities and breath weapon to confuse and scatter enemies. Afterward, they approach foes singly, swooping from the air or fighting on the ground depending on which approach offers the best tactical advantage. Bronzes can also be very patient, watching from afar until a target lets down its guard."
+ trait_creature_bronze_dragon_character_desc:0 "$trait_creature_bronze_dragon_desc$"
```
Comment (robmart) [inline localization/english/wc_traits_l_english.yml:176]: The other creature trait character desc are prefixed with "This character is a (race)"

File: localization/english/wc_traits_l_english.yml [hunk 5]
```diff
@@ -447,15 +450,15 @@
...
  trait_creature_drogbar_character_desc:0 "$trait_creature_drogbar_desc$"
 
  trait_creature_trogg:0 "Trogg"
- trait_creature_trogg_desc:1 "The troggs are a race of brutish, cave-dwelling humanoids, originally descendants of the first type of titan-forged manufactured by the Forge of Wills whose designs proved too complex. The result was the stone-skinned brutes known thereafter as "Troggs." Known for their inferior intelligence, they often come into conflict with their distant relatives, the dwarves, as a result."
+ trait_creature_trogg_desc:1 "The troggs are a race of brutish, cave-dwelling humanoids, originally descendants of the first type of titan-forged manufactured by the Forge of Wills whose designs proved too complex. The result was the stone-skinned brutes known thereafter as "Troggs". Known for their inferior intelligence, they often come into conflict with their distant relatives, the dwarves, as a result."
```
Comment (robmart) [inline localization/english/wc_traits_l_english.yml:461]: Why did you change this? In English the standard is that if a sentence ends with a quote the punctuation is inside the quote.

Also you changed other instances of trogg to lower case, but you missed one here.

**Review by robmart — CHANGES_REQUESTED** [review]

File: localization/english/wc_traits_l_english.yml (hunk 5 above)
Comment (zumbak04) [reply localization/english/wc_traits_l_english.yml:461]: > Why did you change this? In English the standard is that if a sentence ends with a quote the punctuation is inside the quote.

True.

**Review by zumbak04 — APPROVED** [review]
I give it approval, but remind there are @robmart's comments.

File: localization/english/wc_buildings_l_english.yml (hunk 3 above)
Comment (arithon) [reply localization/english/wc_buildings_l_english.yml:7]: Ok, can do that. I just thought that adding the names of the worlds would be better than just "the two worlds".

File: localization/english/wc_traits_l_english.yml (hunk 4 above)
Comment (arithon) [reply localization/english/wc_traits_l_english.yml]: I just added the line we had for some dragons to the other ones here...

File: localization/english/wc_traits_l_english.yml (hunk 5 above)
Comment (arithon) [reply localization/english/wc_traits_l_english.yml:461]: I actually wasn't aware of that rule for quotation and punctuation, you're absolutely right there.

File: localization/english/wc_traits_l_english.yml (hunk 4 above)
Comment (arithon) [reply localization/english/wc_traits_l_english.yml]: However, fixed.

**Review by robmart — APPROVED** [review]
LGTM!
